helm/chaos-mesh/templates/post-install-webhook-job.yaml

```diff
@@ -56,6 +56,23 @@ spec:
                       name: {{ template "chaos-mesh.svc" . }}
                       namespace: {{ .Release.Namespace }}
                       path: "/inject-v1-pod"
+              ---
+              apiVersion: admissionregistration.k8s.io/v1beta1
+              kind: ValidatingWebhookConfiguration
+              metadata:
+                name: {{ template "chaos-mesh.validation" . }}
+              webhooks:
+                - name: {{ template "chaos-mesh.webhook" . }}
+                  clientConfig:
+                    service:
+                      name: {{ template "chaos-mesh.svc" . }}
+                      namespace: {{ .Release.Namespace }}
+                      path: "/validate-v1alpha1-chaos"
+                    caBundle: "${CA_BUNDLE}"
+                  rules:
+                    - operations: [ "CREATE","UPDATE"]
+                      apiGroups: ["pingcap.com"]
+                      apiVersions: ["v1alpha1"]
+                      resources: ["podchaos","iochaos","timechaos","networkchaos"]
```

Comment:
`post-install-webhook-job` is only update the `caBundle`. I don't think we need update the `rules` fields too.

Reply:
good catch